RPerformanceTracking/Private/_RPTClassManipulator+NSURLSessionTask.m

```diff
@@ -53,7 +53,20 @@ void _handleChangedState(NSURLSessionTask *task, NSURLSessionTaskState state)
         uint_fast64_t trackingIdentifier = [objc_getAssociatedObject(task, @selector(_rpt_sessionTask_trackingIdentifier)) unsignedLongLongValue];
         if (trackingIdentifier)
         {
-            [_RPTTrackingManager.sharedInstance.tracker end:trackingIdentifier];
+            NSInteger statusCode = 0;
+            if ([task.response isKindOfClass:[NSHTTPURLResponse class]])
+            {
+                NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *)task.response;
+                statusCode = httpResponse.statusCode;
+            }
+            if (statusCode)
+            {
+                [_RPTTrackingManager.sharedInstance.tracker end:trackingIdentifier statusCode:statusCode];
```

Comment:
I don't think we need the `if (statusCode)` check and we can just always call this unconditionally. 
Even if statusCode is zero (i.e. `statusCode = httpResponse.statusCode` wasn't hit) it won't change the outcome cos zero is the initial value of `measurement.statusCode`.

Reply:
yes.